### DISCUSSION TITLE
GUI: sort by date (doesn't work)

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#4939] [Q&A] [answered]
Stash: 0.26

I just added a video to the stash database it shows following in "Scene >> Details"

![image](https://github.com/stashapp/stash/assets/42092626/fbe49d40-b8d4-445a-83c7-6dbd974cfba7)

In the Scenes page I am listing all videos by Date as follows

![image](https://github.com/stashapp/stash/assets/42092626/d9ca94ac-98a5-4d74-8e53-bf1d24dafce3)

But the recently added scene appears somewhere in the middle on the 3rd page (out of 5 pages as per the above screenshot.

# HELP

How can I arrange the videos such that recently added video to the database show up at the beginning?

**Comment by Ronnie711** [comment]
Changing the option to 'Created At' rather than 'Date' will show the newest added files first.

Date is using the date field in the scene which for the most part is usually the scene date

**Reply by lamyergeier** [reply]
What is meant by scene date?

**Comment by Maista6969** [comment, marked as answer]
Keep in mind that scenes have a Date property that describes when the scene was released: this is what you're currently sorting by

![bilde](https://github.com/stashapp/stash/assets/131594657/1a733014-4525-474e-aa36-46ab019cfd17)
You need to use the `Created At` sort option instead
